README.md

```diff
@@ -8,7 +8,9 @@ python time libraries mocking
 
 ## Usage
 
-`TimeTravel` is context manager patching all time modules currently patched in a single line.
+`TimeTravel` is context manager patching all* time and I\O related modules in a single line.
+
+* All modules currently patched :) .
```

Comment:
This will look like a bullet on the generated page.

- All modules ...

Reply:
Oops.